src/Redux/Book/Book.js

```diff
@@ -1,57 +1,55 @@
-import { v4 as uuidv4 } from 'uuid';
+/* eslint-disable */
 
-const ADDBOOK = './bookStore/Book/ADDBOOK';
-const REMOVEBOOK = './bookStore/Book/REMOVEBOOK';
+// define constant
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const getBook = 'bookstore/books/getBook';
+export const fetchBooks = () => async (dispatch) => {
+  const booksFetch = await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/vBL8ApZ4vcvWCNS7zsXG/books/', {
```

Comment:
All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with `fetch` OR `axios` 🔥

Reply:
I already used the fetch for API calls

Reply:
I know, it was a compliment 😉

Reply:
😂 Ok.. I was wondering what type of fetch again.